drivers/vector/simulator.py

```diff
@@ -45,13 +46,35 @@ def expand_double_gate(gate, i, j, num_qbits):
     return operator
 
 
+def rough_complex(x):
+    re = np.real(x)
+    im = np.imag(x)
+    if re < 0.001 and im < 0.001:
+        return '0'
+    elif re < 0.001:
+        return '%.2f' % im
+    elif im < 0.001:
+        return '%.2f' % re
+    else:
+        return '%.2f+i%.2f' % (re, im)
```

Comment:
Isn't this almost the same as in qmath.py in PR https://github.com/odanielson/QuantumProgramming/pull/27 ?